### PR TITLE
feat(peers): Add peer control plane

### PR DESCRIPTION
**Pull request by leumor**
## Summary
- add Platform API peer control-plane support for peers, including a structured roster plus add, settings, note, and remove mutations backed by the detached runtime SPI ports
- enforce the existing admin full-access plus form-password path for peer mutations, preserve legacy safeguards such as force-removal confirmation semantics, and avoid false add failures when follow-up note writes race
- make the Web Shell peers panel operational with shell-native roster rendering, add-peer, trust/visibility updates, private-note editing, and peer removal while keeping legacy connections toadlets available as fallback
- extend router, toadlet, and Web Shell test coverage for the new peer-control-plane routes and edge cases, including force removal, blank note handling, opennet add validation, and effective routing-disabled reporting

## Testing
- ./gradlew :platform-api:test
- ./gradlew :platform-web-shell:test
- ./gradlew :adapter-http-legacy-admin:test
- ./gradlew :test --tests 'network.crypta.platform.api.PlatformApiRouterTest'
- ./gradlew :platform-web-shell:test
- ./gradlew test

### CODE REVIEW

File: platform-web-shell/src/main/resources/network/crypta/platform/webshell/static/web-shell.js [hunk 1]
```diff
@@ -215,14 +234,219 @@
...
+
+    card.append(definitionList(peerDetailEntries(peer)));
+
+    const links = document.createElement("div");
+    links.className = "peer-card-links";
+    const rawLink = document.createElement("a");
+    rawLink.className = "button button-secondary";
+    rawLink.href = apiUrl(
+      `peers/${encodeURIComponent(peer.identity)}?includeMetadata=true&includeVolatile=true`,
+    );
```
[the remark this answers is not in the document]
Comment (leumor) [reply platform-web-shell/src/main/resources/network/crypta/platform/webshell/static/web-shell.js:436]: Fixed in `08a617be80`.

The CodeQL annotation is attached to a shifted line in the PR diff, but the underlying issue was real: the queue snapshot render path in `web-shell.js` was still mounting `contentHtml` via `innerHTML`.

I replaced that path with an inert `DOMParser`-based sanitizer before the fragment is attached to the live DOM. The sanitizer now strips executable elements, inline event handlers, `style`/`srcdoc`, and unsafe URL-bearing attributes while preserving the existing queue link rewriting and form-password injection flow.

Validation after the fix:
- `./gradlew :platform-web-shell:test`
- `./gradlew test`

Both passed.

File: platform-api/src/main/java/network/crypta/platform/api/PlatformApiRouter.java [hunk 2]
```diff
@@ -421,14 +421,77 @@ private PlatformApiResponse routeNodeRequest(List<String> segments, PlatformApiR
...
+   * @param resource second path segment beneath {@code /peers}
+   * @param request full request metadata, including query parameters
+   * @return JSON response for the selected peer endpoint
+   */
+  private PlatformApiResponse routePeerResource(String resource, PlatformApiRequest request) {
+    if ("roster".equals(resource)) {
+      if (!"GET".equals(request.method())) {
+        return methodNotAllowed("GET", GET_ONLY_MESSAGE);
+      }
+      return PlatformApiResponse.ok(peersApiHandler.roster());
```
[the remark this answers is not in the document]
Comment (leumor) [reply platform-api/src/main/java/network/crypta/platform/api/PlatformApiRouter.java]: Fixed in `158cec4a5e`.

The regression was real: reserving `GET /peers/roster` ahead of the legacy peer lookup broke `GET /peers/{id}` for identifiers such as `roster`, and `GET /peers/add` had the same problem for `add`.

I changed the shell-facing structured roster route to `GET /peers/summary` and restored the old `GET /peers/{nodeIdentifier}` fallback behavior for `roster` and `add`. `POST /peers/add` remains the only special add route. The Web Shell now loads from `/peers/summary`, and router coverage was extended for the compatibility cases.

Validation after the fix:
- `./gradlew :test --tests 'network.crypta.platform.api.PlatformApiRouterTest'`
- `./gradlew :platform-web-shell:test`
- `./gradlew test`

All passed.

File: platform-api/src/main/java/network/crypta/platform/api/PlatformApiRouter.java [hunk 3]
```diff
@@ -421,14 +421,76 @@ private PlatformApiResponse routeNodeRequest(List<String> segments, PlatformApiR
...
+   * @param resource second path segment beneath {@code /peers}
+   * @param request full request metadata, including query parameters
+   * @return JSON response for the selected peer endpoint
+   */
+  private PlatformApiResponse routePeerResource(String resource, PlatformApiRequest request) {
+    if ("summary".equals(resource)) {
+      if (!"GET".equals(request.method())) {
+        return methodNotAllowed("GET", GET_ONLY_MESSAGE);
+      }
+      return PlatformApiResponse.ok(peersApiHandler.roster());
```
[the remark this answers is not in the document]
Comment (leumor) [reply platform-api/src/main/java/network/crypta/platform/api/PlatformApiRouter.java]: Fixed in `2e302cbb7f`.

The regression was real: moving the shell-facing roster to `GET /peers/summary` still reserved a second path segment and therefore broke legacy `GET /peers/{nodeIdentifier}` lookups for identifiers such as `summary`.

I moved the shell-facing summary view onto the collection endpoint instead: `GET /peers?view=summary`. That keeps `GET /peers/{nodeIdentifier}` fully backward-compatible for `summary`, `roster`, `add`, and any other legacy-resolved identifier, while still giving the Web Shell a dedicated structured roster view.

Validation after the fix:
- `./gradlew :test --tests 'network.crypta.platform.api.PlatformApiRouterTest'`
- `./gradlew :platform-web-shell:test`
- `./gradlew test`

All passed.
